### PR TITLE
feat: Emoji Status Control in Raise Hand Button

### PR DESCRIPTION
**Pull request by Gigate**
### What does this PR do?
This PR adds a dropdown menu, which allows a user to change their emoji status, to the raise hand button. 

### Motivation
In some lectures professors like to use the status for getting a general overview over the students opinions. This is more comfortable than starting a poll. The menu to change the status was pretty hidden. This PR makes it easier to access it.

### CODE REVIEW

**Comment by MaximKhlobystov** [comment]
Hi Gigate,
Thank you for this contribution! Could you please confirm if you already sent in the signed Contributor License Agreement? See https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code.
Thanks in advance!

**Comment by antobinary** [comment]
@Gigate signed a CLA last year while working on https://github.com/bigbluebutton/bigbluebutton/pull/13634#issuecomment-965504592

**Comment by MaximKhlobystov** [comment]
> @Gigate signed a CLA last year while working on [#13634 (comment)](https://github.com/bigbluebutton/bigbluebutton/pull/13634#issuecomment-965504592)

Good. Thanks!

**Comment by steffen-moser** [comment]
Is there any chance to get this merged into BBB-2.6? Is there anything we can/should do from our side?

File: bigbluebutton-html5/imports/ui/components/actions-bar/raise-hand/component.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import withShortcutHelper from '/imports/ui/components/shortcut-help/service';
+import BBBMenu from '/imports/ui/components/common/menu/component';
+import Button from '/imports/ui/components/common/button/component';
+import Styled from './styles';
+
+const propTypes = {
+  intl: PropTypes.shape({
+    formatMessage: PropTypes.func.isRequired,
+  }).isRequired,
+  shortcuts: PropTypes.string,
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/ui/components/actions-bar/raise-hand/component.jsx:13]:
```suggestion
  shortcuts: PropTypes.objectOf(PropTypes.string),
```

File: bigbluebutton-html5/imports/ui/components/actions-bar/raise-hand/component.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+};
+
+const defaultProps = {
+  shortcuts: '',
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/ui/components/actions-bar/raise-hand/component.jsx:17]:
```suggestion
  shortcuts: [],
```

**Pull request merged**